app/models/participant.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class Participant < ApplicationRecord
+  belongs_to :unit
+
+  validates_uniqueness_of :pbs_id, allow_blank: true
+
+  enum gender: { male: 'male', female: 'female' }
+
+  def full_name
+    "#{first_name} #{last_name} v/o #{scout_name}"
```

Comment:
what if the user has no scoutname?
what about: `[first_and_last_name, scout_name].join(" v/o ")`